### DISCUSSION TITLE
Integrating third party clips / events (Reolink)

### DISCUSSION THREAD

**Discussion by phillee** [blakeblackshear/frigate#14345] [Ask A Question] [answered]
I have battery + solar Reolinks (Argus Track) in remote locations. Due to the battery limitations, I cannot do continuous recording. 

I have Amcrest POE cams for the rest of my setup with Frigate which works great. The battery-powered Reolinks have built in motion detection on-device, which are then collected by the Reolink Home Hub including clips. Alternatively I think I can do on-device recording, but the Home Hub seems to simplify some of the collection. 

I'd like to pipe the Reolink clips and events into Frigate. Alternatively I've considered using Home Assistant as the aggregator, but I'm a big fan of the Frigate UI after the recent improvements. 

Looking for recommendations on the ideal setup.

**Comment by hawkeye217** [comment, marked as answer]
Frigate's object tracking, review item management, and recordings maintenance all require access to a live video source. There's no way to manually feed other video clips into Frigate in a way that wouldn't require you to edit your config file for every new clip. You could [use the API to create manual events](https://docs.frigate.video/integrations/api#post-apieventscamera_namelabelcreate), but unfortunately that requires Frigate to have access to your camera.

Your use case isn't something Frigate itself is designed to support, unfortunately.

**Reply by phillee** [reply]
Thanks. Any recommendations if I want the Frigate-like timeline / UI with the ability to support multiple event sources? 

I understand the core of Frigate is what you mentioned. With the recent UI updates it feels like the best open source UI for camera timelines and I'd like to be able to use it as my primary interface.

**Reply by hawkeye217** [reply]
Unfortunately I don't have any recommendations at this time. Frigate would need to be extended significantly to support that kind of use case.
